### PR TITLE
ADCM-2417 fix form configuration

### PR DESCRIPTION
**Pull request by drKaramazin**

### CODE REVIEW

File: web/src/app/shared/configuration/services/field.service.ts [hunk 1]
```diff
@@ -317,9 +317,9 @@ export class FieldService {
   }
 
   checkValue(value: resultTypes, type: TNForm): resultTypes {
-    if ((value === '' || isEmptyObject(value)) && type === 'map') return {};
+    if ((value === '' || isEmptyObject(value) || value === null) && type === 'map') return {};
```
Comment (remizov-arena) [inline web/src/app/shared/configuration/services/field.service.ts:320]: May be will be better ? And not recalc and rewrite conditions
```
if (value === '' || value === null || isEmptyObject(value)) {
  if (type === 'map') return {};
  if (type === 'list') return [];
  return null;
} 
```

**Review by remizov-arena — DISMISSED** [review]

**Comment by drKaramazin** [comment]
bender build

**Comment by BenderArenadata** [comment]
Hey, @drKaramazin! Build has been launched!

**Comment by a-alferov** [comment]
bender build

**Comment by BenderArenadata** [comment]
Hey, @a-alferov! Build has been launched!

**Review by remizov-arena — APPROVED** [review]

**Pull request merged**